dashboard/src/app/dashboard/[dashboardId]/web-vitals/interactiveWebVitalsChartSection.tsx

```diff
@@ -0,0 +1,230 @@
+      chartColor: 'var(--chart-2)',
+      isActive: active === 'LCP',
+      onClick: () => setActive('LCP'),
+    },
+    {
+      title: (
+        <span>
+          Interaction to Next Paint
+          <P75Badge />
+        </span>
+      ),
+      value: formatCardValue('INP', summary.inpP75),
+      footer:
+        summary.inpP75 === null ? null : (
+          <div style={{ color: getCwvStatusColor('INP', summary.inpP75) }}>
+            <CoreWebVitalBar metric='INP' value={summary.inpP75} thresholds={CWV_THRESHOLDS} />
+          </div>
+        ),
+      chartColor: 'var(--chart-3)',
+      isActive: active === 'INP',
+      onClick: () => setActive('INP'),
+    },
+    {
+      title: (
+        <span>
+          First Contentful Paint
+          <P75Badge />
+        </span>
+      ),
+      value: formatCardValue('FCP', summary.fcpP75),
+      footer:
+        summary.fcpP75 === null ? null : (
+          <div style={{ color: getCwvStatusColor('FCP', summary.fcpP75) }}>
+            <CoreWebVitalBar metric='FCP' value={summary.fcpP75} thresholds={CWV_THRESHOLDS} />
+          </div>
+        ),
+      chartColor: 'var(--chart-4)',
+      isActive: active === 'FCP',
+      onClick: () => setActive('FCP'),
+    },
+    {
+      title: (
+        <span>
+          Time to First Byte
+          <P75Badge />
+        </span>
+      ),
+      value: formatCardValue('TTFB', summary.ttfbP75),
+      footer:
+        summary.ttfbP75 === null ? null : (
+          <div style={{ color: getCwvStatusColor('TTFB', summary.ttfbP75) }}>
+            <CoreWebVitalBar metric='TTFB' value={summary.ttfbP75} thresholds={CWV_THRESHOLDS} />
+          </div>
+        ),
+      chartColor: 'var(--chart-5)',
+      isActive: active === 'TTFB',
+      onClick: () => setActive('TTFB'),
+    },
+  ];
+
+  const titles: Record<CoreWebVitalName, string> = {
+    CLS: 'Cumulative Layout Shift',
+    LCP: 'Largest Contentful Paint',
+    INP: 'Interaction to Next Paint',
+    FCP: 'First Contentful Paint',
+    TTFB: 'Time to First Byte',
+  } as const;
+
+  const chartData = seriesByMetric[active] || [];
+
+  function formatThreshold(metric: CoreWebVitalName, value: number): string {
+    if (metric === 'CLS') return value.toString();
+    return formatShortFromMilliseconds(value);
+  }
+
+  const referenceLines = CWV_THRESHOLDS[active]?.map((y, idx) => {
+    const label = `${idx === 0 ? 'Good' : 'Needs improvement'} (≤ ${formatThreshold(active, y)})`;
+    const stroke = idx === 0 ? 'var(--cwv-threshold-good)' : 'var(--cwv-threshold-ni)';
+    return { y, label, stroke, strokeDasharray: '6 6', labelFill: stroke };
+  });
+
+  const SERIES_DEFS: ReadonlyArray<MultiSeriesConfig> = [
+    { dataKey: 'value.0', stroke: 'var(--cwv-p50)', name: 'P50' },
+    { dataKey: 'value.1', stroke: 'var(--cwv-p75)', name: 'P75' },
+    { dataKey: 'value.2', stroke: 'var(--cwv-p90)', name: 'P90' },
+    { dataKey: 'value.3', stroke: 'var(--cwv-p99)', name: 'P99' },
+  ] as const;
```

Comment:
Suggestion:
- Move the functions without dependencies out of the function to declutter
- Move constants outside of component for performance / decluttering
- If `referenceLines` is expensive to compute, wrap in useMemo